src/base/loaders/I3DMLoaderBase.js

```diff
@@ -84,13 +84,13 @@ export class I3DMLoaderBase extends LoaderBase {
 
 		} else {
 
-			const externalUri = this.resolveExternalURL( arrayToString( bodyBytes ) );
-			promise = fetch( externalUri, this.fetchOptions )
+			this.workingPath = this.resolveExternalURL( arrayToString( bodyBytes ) );
```

Comment:
I'd prefer not to overwrite the existing working path for the I3DMLoader since this is intended to be the working path for the I3DM path being loaded, not the GLTF. I think we can pass the GLTF working path in via the `parse` constructor.

Reply:
I have difficulties to follow, do you mean this `parse` ?
https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/564fb9faa096292dec3b7d2ca00dceb75e7af9b1/src/three/loaders/I3DMLoader.js#L73

Should I return it  here ? 
https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/564fb9faa096292dec3b7d2ca00dceb75e7af9b1/src/base/loaders/I3DMLoaderBase.js#L110-L116